### PR TITLE
Fix release of single arch images to GHCR

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

Closes https://github.com/tektoncd/plumbing/issues/2221

Add missing auth for ghcr.io
/kind misc

Verified with `ko-gcloud`: https://github.com/tektoncd/plumbing/pkgs/container/plumbing%2Fko-gcloud/312396764?tag=v20241126-66f4b045c7

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/resources/images/single-arch-template.yaml [hunk 1]
```diff
@@ -76,6 +76,12 @@ spec:
               DESTINATIONS=""
               for tag in ${TAGS}; do DESTINATIONS="${DESTINATIONS} --destination ${IMAGE_URL}:${tag}"; done
               IFS=$OIFS
+
+              IMAGE_REGISTRY_USER=tekton-robot
+              IMAGE_REGISTRY_TOKEN=$(cat $IMAGE_REGISTRY_CREDENTIALS)
```
Comment (afrittoli) [inline tekton/resources/images/single-arch-template.yaml:81]: NIT: $(cat "$IMAGE_REGISTRY_CREDENTIALS") is safer

**Review by afrittoli — COMMENTED** [review]
Thanks, just a minor comment

File: tekton/resources/images/single-arch-template.yaml (hunk 1 above)
Comment (AlanGreene) [reply tekton/resources/images/single-arch-template.yaml]: 👍 updated

**Review by afrittoli — APPROVED** [review]
/lgtm
/approve

**Pull request merged**